### ISSUE TITLE
Flight model of the H135

### ISSUE THREAD

**Issue by Leonhard96L** [Leonhard96L/NVGSIM#12]

**Comment by Leonhard96L** [comment]
One important point:
The seat of the H135 helicopter is moveable. You can move the seat forward and backward. In your NVG Sim there was just the pedal adjustable. Would it maybe make more sense to build a moveable seat? Or is this to complex?

**Comment by Leonhard96L** [comment]
Hi Ivan we took the measurment from our H135 Sim:
As reference we took the eyepoint which we projected on the Cockpit floor. (Yellow tape)

**Comment by Leonhard96L** [comment]
![Image](https://github.com/Leonhard96L/NVGSIM/assets/134842111/45ad9908-a856-4cc4-8016-5672dca000e4)



![Image](https://github.com/Leonhard96L/NVGSIM/assets/134842111/86b1af99-7efe-42d8-b50c-82cea4b047bc)

Distance from the eyepoint to the middle of the cyclic is 36,6 cm

**Comment by Leonhard96L** [comment]
Distance from eypoint to the pedal joint: 85,5cm


![Image](https://github.com/Leonhard96L/NVGSIM/assets/134842111/c7bf4a2b-5e6e-4bed-bbd4-cd779edb9d12)



![Image](https://github.com/Leonhard96L/NVGSIM/assets/134842111/6b1b4de1-993e-4d50-b0c8-9b863a9b9202)

**Comment by Leonhard96L** [comment]
Distance from eyepoint to the pedal step: 90cm

![Image](https://github.com/Leonhard96L/NVGSIM/assets/134842111/c2005a92-fa92-41d4-a91d-c2277cdbf081)
![Image](https://github.com/Leonhard96L/NVGSIM/assets/134842111/0e94d74b-d952-4394-93db-0431c129adb0)

**Comment by Leonhard96L** [comment]
Distance from eyepoint to the joint of the collective: 25,3 cm


![Image](https://github.com/Leonhard96L/NVGSIM/assets/134842111/0d7f1ce2-6b8f-4d75-a79f-0b2639a58d43)
![Image](https://github.com/Leonhard96L/NVGSIM/assets/134842111/958efa0a-bb7d-46b2-ba93-b4e4dc489381)

**Comment by Leonhard96L** [comment]
Distance from the middle of the seat to the middle of the collective: 29,3 cm 


![Image](https://github.com/Leonhard96L/NVGSIM/assets/134842111/a8a86994-3671-47c2-a899-84ac9352e036)
![Image](https://github.com/Leonhard96L/NVGSIM/assets/134842111/10007fac-484b-4130-a14b-4abdd235c1ff)

**Comment by Leonhard96L** [comment]
Distance from the Cyclic to the slant console: 22,8cm

![Image](https://github.com/Leonhard96L/NVGSIM/assets/134842111/883df3c4-1160-486a-8edb-81ad820b52bc)
![Image](https://github.com/Leonhard96L/NVGSIM/assets/134842111/d0339b0b-8ed5-44af-978e-14cb459adf1e)

**Comment by Leonhard96L** [comment]
Distance from cockpitfloor to seat: approx. 28cm 

![Image](https://github.com/Leonhard96L/NVGSIM/assets/134842111/d108e99f-41b2-4771-86da-3a18a00e1f62)
![Image](https://github.com/Leonhard96L/NVGSIM/assets/134842111/0a72f087-7597-4f6a-93a2-7aefb727e75d)

**Comment by miletovic** [comment]
> One important point: The seat of the H135 helicopter is moveable. You can move the seat forward and backward. In your NVG Sim there was just the pedal adjustable. Would it maybe make more sense to build a moveable seat instead of movable pedals? Or is this to complex?

I discussed this with Stephen. For the "new" Bruner pedals in the NVG simulator, it is actually easier to build it such that the seat is adjustable. Can you confirm that this is what you want? If so, than I will ask Stephen to include a movable seat in the design of the platform instead of movable pedals.

**Comment by Leonhard96L** [comment]
Yes i will ask Michael Mayrhofer and give you the answer a.s.a.p!

**Comment by miletovic** [comment]
That's great, I'll tell Stephen!
